components/brave_vpn/brave_vpn_resources.h

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_VPN_BRAVE_VPN_RESOURCES_H_
+#define BRAVE_COMPONENTS_BRAVE_VPN_BRAVE_VPN_RESOURCES_H_
+
+#include "base/no_destructor.h"
+#include "components/grit/brave_components_strings.h"
+#include "ui/base/webui/web_ui_util.h"
+
+namespace brave_vpn {
+
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
why are we including this file at all for android? Also this file should be brave/browser/ui/webui/brave_vpn/brave_vpn_localized_string_provider.h and use a method `AddLocalizedStrings` - see https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/certificate_manager_localized_strings_provider.h?q=certificate_manager_localized_strings_provider&ss=chromium

Reply:
done